react-router-redux/react-router-redux-tests.ts

```diff
@@ -6,15 +6,10 @@
 
 import { createStore, combineReducers, applyMiddleware } from 'redux';
 import { browserHistory } from 'react-router';
-import { syncHistory, routeReducer } from 'react-router-redux';
+import { syncHistoryWithStore, routerReducer } from 'react-router-redux';
 
-const reducer = combineReducers({ routing: routeReducer });
+const reducer = combineReducers({ routing: routerReducer });
+const store = createStore(reducer);
 
-// Sync dispatched route actions to the history
-const reduxRouterMiddleware = syncHistory(browserHistory);
-const createStoreWithMiddleware = applyMiddleware(reduxRouterMiddleware)(createStore);
-
-const store = createStoreWithMiddleware(reducer);
-
-// Required for replaying actions from devtools to
-reduxRouterMiddleware.listenForReplays(store);
+// Create an enhanced history that syncs navigation events with the store
+const history = syncHistoryWithStore(browserHistory, store)
```

Comment:
I think we have to add some more tests to cover `push(location)`, `replace(location)`, `go(number)`, `goBack()`, `goForward()`, etc... Don't you think so?